content/tips-on-applying-for-teacher-training.md

```diff
@@ -0,0 +1,180 @@
+criminal convictions. Training providers can advise you whether it’ll affect
+your application.
+
+## Prepare your application
+
+You can [find and apply to most postgraduate teacher training courses on GOV.UK](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+
+If you do not find what you were looking for, there are [other ways to train](/ways-to-train).
+
+Before you start, it’s particularly important to think about your referees and
+personal statement.
+
+Some people do not get on the course they want because their application has
+not convinced the admissions team that they’re right for it.
+
+Talk to an experienced teacher for help preparing your application.
+
+## Choose your referees
+
+You need 2 references to apply.
+
+Unlike a standard work reference, your references for teacher training need to
+be named individuals rather than (for example) a human resources team or
+academic department.
+
+Your referees will be asked to write 500 words about your character and
+suitability for teaching.
+
+They’ll have 2 weeks to respond, so it’s worth contacting them before you apply
+so they understand why you’re applying and what they’ll be asked to do.
+
+Choose referees who’ll endorse your:
+
+* teaching passion and potential
+* suitability for working with children
+* academic abilities
+* reliability and professionalism
+
+Ideal referees could include:
+
+* your university tutor or supervisor
+* your current line manager at work
+* your previous employer
+* a teacher at a school where you work or volunteer
+* a supplier or client you’ve worked with (if you’re self-employed)
+
+Referees should not be family members, partners or friends.
+
+Training providers will only accept a character reference as a second reference.
+
+It’s important to have at least one academic or professional reference.
+
+If you’re applying for a salaried course, one of your references must be from
+an employer.
+
+If you’re struggling, get free one-on-one support with choosing your referees.
+
+## Your personal statement
+
+Explaining why you want to be a teacher is the crux of your application - so
+it’s worth taking your time on it.
+
+A personal statement is usually about 500 words. It’s your chance to show your
+motivation, commitment and teaching potential.
+
+* You may want to cover some of the following:
+* why you want to teach
+* what inspired you to choose teaching
+* your thoughts on children and young children’s wellbeing and the education system
+* your understanding of the demands and rewards of the profession
+* personal qualities that will make you a good teacher
+* your contribution to the life of the school outside the classroom – for
+  example, running extra-curricular activities and clubs
+* details any experience you have working with children and what you learnt
+* your suitability to teach a subject or group
+* evidence of the knowledge you have in your subject
+
+Think about <mark>who and what to teach</mark> before you apply.
```

Comment:
this marked text, 'who and what to teach', shows up on the site as highlighted yellow text - is that intended? Do we use that style elsewhere on site?

Reply:
I've fixed that already, it was a link to a page that doesn't yet exist (and now won't) - the line has been removed.

The `<mark>` just makes it yellow so it's clear something needs to be done!